source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -355,3 +355,36 @@
 		opacity: 0.5;
 	}
 }
+
+.is-style-brush-stroke {
+	position: relative;
+
+	&::before {
+		content: "";
+		min-height: var(--wp--custom--brush-stroke--spacing--height);
+		position: absolute;
+		top: 100%;
+		left: 0;
+		right: 0;
+		width: auto;
+		mask-image: url(../images/brush-stroke-mask.svg);
+		mask-repeat: no-repeat;
+		mask-size: cover;
+		mask-position: bottom;
+		background-color: inherit;
+	}
+}
+
+@media (min-width: 890px) {
+	// Only add this for pages with the block on it.
+	html:has(main > .is-style-brush-stroke) {
```

Comment:
caniuse puts [:has support](https://caniuse.com/?search=has) pretty low, compared to what we usually try to support (no Firefox support at all 😞), but I think this can be considered a progressive enhancement. 

Open to other suggestions though.